packages/story-editor/src/components/panels/design/videoAccessibility/videoAccessibility.js

```diff
@@ -171,6 +180,7 @@ function VideoAccessibilityPanel({ selectedElements, pushUpdate }) {
           value={poster}
           cropParams={cropParams}
           onChange={handleChangePoster}
+          onDelete={onDelete}
```

Comment:
Here the Delete is essentially the same as Reset now, right? Since it's not actually fully deleted. Or in case of Delete, the image is actually deleted from the library as well, in case of Reset not?

Reply:
Is it worth renaming `onDelete` to something else to make it less confusing?

Reply:
How about onDeleteMedia and onUploadMedia?